src/wavesurfer.js

```diff
@@ -26,6 +26,10 @@ import PeakCache from './peakcache';
  * initialized ScriptProcessorNode or leave blank.
  * @property {boolean} autoCenter=true If a scrollbar is present, center the
  * waveform around the progress
+ * @property {number} autoCenteringRate=5 If autoCenter is active, rate at which the
+ * waveform is centered
+ * @property {boolean} autoCenterImmediately=false If autoCenter is active, immediately
+ * center waveform around progress
```

Comment:
not sure what 'around progess' means. either re-word or remove that part.

Reply:
I just took over the wording from the original 'autoCenter' option, but I can change that to make it clearer.

Reply:
ok. you changed the docs but not the actual code as well.